package.json

```diff
@@ -9,17 +9,19 @@
     }
   },
   "dependencies": {
+    "lodash": "~2.4.1",
+    "q": "~0.9.7",
+    "vumigo_v01": "~0.1.22",
+    "vumigo_v02": "~0.2"
+  },
+  "optionalDependencies": {
```

Comment:
Hmm. Failure to install optional dependencies doesn't fail the install process, which might be annoying -- https://docs.npmjs.com/files/package.json#optionaldependencies.

Just mentioning this in case others feel it's a show stopper.